### PR TITLE
Upgrade csv

### PR DESCRIPTION
**Pull request by mateo0203**
test

### CODE REVIEW

**Review by farisdurrani — APPROVED** [review]

**Comment by farisdurrani** [comment]
All good for me. Just fix the conflicts and I recommend waiting for Karthik's approval

**Comment by karkir0003** [comment]
wait a sec. i dont think mateo finished this task

**Review by karkir0003 — CHANGES_REQUESTED** [review]
lmk when you have this feature ready

File: frontend/playground-frontend/src/components/Home/CSVInputFile.js
Comment (karkir0003) [inline frontend/playground-frontend/src/components/Home/CSVInputFile.js:48]: @mateo0203 lmk when you resolved the blocker with daniel

**Review by karkir0003 — CHANGES_REQUESTED** [review]
wait on mateo to resolve blocker with daniel

**Comment by farisdurrani** [comment]
Closing since issue is resolved in #552 using this PR's code

**Pull request closed without merging**
